utils/interfaces/_logs.py

```diff
@@ -14,6 +14,10 @@
 from utils.interfaces._core import InterfaceValidator
 from utils._context.component_version import ComponentVersion
 
+import pydevd_pycharm
+
+pydevd_pycharm.settrace("localhost", port=5555, stdoutToServer=True, stderrToServer=True, suspend=False)
```

Comment:
I'm a little bit afraid to have this everywhere (local laptops on mac/windows/linux, CI ...)